<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summaryPage</class>
 <widget class="QWidget" name="summaryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>750</height>
   </rect>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>751</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>929</width>
      <height>749</height>
     </rect>
    </property>
    <widget class="QLabel" name="sharedMeatLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jaetut lihat ryhmittäin</string>
     </property>
    </widget>
    <widget class="QWebEngineView" name="sankeyWebEngineView" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>901</width>
       <height>481</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
    <widget class="QTableWidget" name="groupSummaryTableWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>321</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="meatSharedTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="groupSummarylabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ryhmän tiedot</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
